ceph-salt-formula/salt/ceph-salt/cephorch.sls

```diff
@@ -0,0 +1,13 @@
+{% import 'macros.yml' as macros %}
+
+{{ macros.begin_stage('Add host to ceph orchestrator') }}
+
+{% set bootstrap_host = pillar['ceph-salt']['bootstrap_minion'].split('.', 1)[0] %}
+
+add host to ceph orch:
+  cmd.run:
+    - name: |
+        ssh -o "StrictHostKeyChecking=no" -i /tmp/ceph-salt-ssh-id_rsa root@{{ bootstrap_host }} "ceph orch host add {{ grains['host'] }}"
```

Comment:
Can we do this on an admin host instead? What happens if this minion is no longer part of the cluster?

Reply:
Also, even when this is done on an admin host, the command `ceph orch host add` will fail if and when `ceph-common` is no longer installed on the host...

Reply:
@sebastian-philipp I've pushed a new commit that will not rely on the bootstrap minion to get ceph `config`+`keyring` and to run `ceph orch host add`.

Reply:
@smithfarm Correct. That is something to be addressed during the implementation of https://github.com/ceph/ceph-salt/issues/120